.eslintrc

```diff
@@ -21,6 +21,7 @@
       "react"
     ],
     "rules": {
-      "react/jsx-filename-extension": 0
+      "react/jsx-filename-extension": 0,
+      "import/no-extraneous-dependencies": ["error", {"devDependencies": true}]
```

Comment:
Isn't this the default behavior? [Link](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md#options)

Why adding this rule explicitly then?

Reply:
When importing devDependencies it is giving a lint error. This allows importing devDependencies.

Reply:
The proper solution is to make a `.eslintrc` in the `test` folder and add this rule in it. Don't add this rule to the global `.eslintrc` file. [Link](https://github.com/airbnb/javascript/issues/959#issuecomment-234318804)

Reply:
The [proper solution](https://github.com/airbnb/javascript/issues/959#issuecomment-234318804) is to make a `.eslintrc` in the `test` folder and add this rule in it. Don't add this rule to the global `.eslintrc` file.

Reply:
I think we will be importing dev-dependencies on both server and client. Creating two more eslintrc for just this rule is an overhead.